Extend/Core/Email.php

```diff
@@ -89,7 +89,7 @@ private function _sendEmailWithOrderLanguage($order, $subject, $function)
         $oOldShop = $this->_getShop();
         $iOldTplLang = $oLang->getTplLanguage();
         $iOldBaseLang = $oLang->getTplLanguage();
-        $iOrderLanguage = $order->oxorder__oxlang->value;
+        $iOrderLanguage = $oOrder->oxorder__oxlang->value;
```

Comment:
is there a getter for it?

Reply:
What do you mean? This is already using OXID's magic getter functionality.

Reply:
Explanation accepted. Looking through the documentation and examples by oxid this is really the right approach.